packages/terra-clinical-item-view/tests/nightwatch/item-view-spec.js

```diff
@@ -1,47 +1,58 @@
+/* eslint-disable no-unused-expressions */
 // eslint-disable-next-line import/no-extraneous-dependencies
 const resizeTo = require('terra-toolkit/lib/nightwatch/responsive-helpers').resizeTo;
 
 module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
   'Displays a clinical item view with default props': (browser) => {
     browser.url(`${browser.launchUrl}/#/tests/item-view-tests/default`);
-    browser.assert.elementPresent('#ItemView');
+    browser.expect.element('#ItemView').to.be.present;
```

Comment:
could remove the no-unused-expressions at the top for the linter and add a .to.equal(true) after each line of checking whether the element is present

browser.expect.element('#ItemView').to.be.present.to.equal(true)

Reply:
I prefer disabling the lint rule.